src/safe_apps/models.py

```diff
@@ -1,6 +1,17 @@
+from enum import Enum
+
 from django.contrib.postgres.fields import ArrayField
+from django.core.validators import RegexValidator
 from django.db import models
 
+HostnameValidator = RegexValidator(
```

Comment:
Since this is a constant we should follow PEP8 conventions for the naming https://www.python.org/dev/peps/pep-0008/#constants

Suggestion: `_HOSTNAME_VALIDATOR`

Reply:
I'm curious how I got through that Linter Castle...